benchmark/single-source/IndexPathTest.swift

```diff
@@ -0,0 +1,167 @@
+//===--- IndexPathTest.swift -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+import Foundation
+
+let size = 200
+let tags: [BenchmarkCategory] = [.validation, .api, .IndexPath]
+
+public let IndexPathTest = [
+  BenchmarkInfo(name: "IndexPathSubscriptMutation", 
+  runFunction: { n in run_IndexPathSubscriptMutation(n * 1000, size)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathSubscriptRangeMutation", 
+  runFunction: { n in run_IndexPathSubscriptRangeMutation(n * 1000, size)}, 
+  tags: tags),
+
+  BenchmarkInfo(name: "IndexPathMaxBeginning", 
+  runFunction: { n in run_IndexPathMaxBeginning(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMaxMiddle", 
+  runFunction: { n in run_IndexPathMaxMiddle(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMaxEnd", 
+  runFunction: { n in run_IndexPathMaxEnd(n * 1000)}, 
+  tags: tags),
+
+  BenchmarkInfo(name: "IndexPathMinBeginning", 
+  runFunction: { n in run_IndexPathMinBeginning(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMinMiddle", 
+  runFunction: { n in run_IndexPathMinMiddle(n * 1000)}, 
+  tags: tags),
+  BenchmarkInfo(name: "IndexPathMinEnd", 
+  runFunction: { n in run_IndexPathMinEnd(n * 1000)}, 
+  tags: tags),
+]
+
+@inline(__always)
+func indexPath(_ size: Int, 
+  reversed: Bool = false) -> IndexPath {
```

Comment:
Standard library style is to have these all on one line if it fits within 80 columns; if it doesn't, then the parameters can be on their own line like so:

```swift
func foo(
  bar: VeryVeryLongTypeName,
  baz: VeryVeryLongTypeName,
  boo: VeryVeryLongTypeName
) -> Int {
  ...
}
```

Indents are usually two spaces.

Reply:
Thanks! Made the changes.